geode-core/src/main/java/org/apache/geode/internal/cache/wan/serial/SerialGatewaySenderQueue.java

```diff
@@ -496,13 +489,19 @@ void peekEventsFromIncompleteTransactions(List<AsyncEvent<?, ?>> batch, long las
         }
       }
       if (incompleteTransactionIdsInBatch.size() == 0 ||
-          retries++ == GET_TRANSACTION_EVENTS_FROM_QUEUE_RETRIES) {
+          retries >= sender.getRetriesToGetTransactionEventsFromQueue()) {
         break;
       }
+      retries++;
+      try {
+        Thread.sleep(GET_TRANSACTION_EVENTS_FROM_QUEUE_WAIT_TIME_MS);
+      } catch (InterruptedException e) {
+        Thread.currentThread().interrupt();
+      }
```

Comment:
Same comment here.

GET_TRANSACTION_EVENTS_FROM_QUEUE_WAIT_TIME_MS is 1ms by default.

GET_TRANSACTION_EVENTS_FROM_QUEUE_RETRIES is 3 by default.

So, thats 3 ms of retries by default. I'm not sure thats going to be enough.

Reply:
ok. I will set it to 10 as suggested by @gesterzhou